minorminer/utils/parallel_embeddings.py

```diff
@@ -333,8 +449,52 @@ def find_sublattice_embeddings(
             "source graphs must a graph constructed by "
             "dwave_networkx as chimera, pegasus or zephyr type"
         )
+    if tile_embedding is not None:
+        if not _is_valid_embedding(tile_embedding, S, tile, one_to_iterable):
+            raise ValueError("tile_embedding is invalid for S and tile")
+        else:
+            use_filter = False  # Unnecessary
```

Comment:
Minor, but else not required here.
```suggestion

        use_filter = False  # Unnecessary
```